src/app/renderer/actions/fileTree/resetFilter.ts

```diff
@@ -0,0 +1,8 @@
+import {container} from '../../container';
+import {ResetFilter} from '../../../../domain/useCases/fileTree/ResetFilter';
+
+const useCase: ResetFilter = container.get(ResetFilter);
+
+export function resetFilter() {
```

Comment:
should have return type `: Promise<void>`

Reply:
done